config/licenseHeaderRaw

```diff
@@ -0,0 +1,13 @@
+Copyright 2022 The Sigstore Authors.
```

Comment:
could this be `$YEAR` instead of `2022`?

Reply:
I need to support that in Autostyle first. Hopefully, there are enough days left in 2022 for that.